daemon/entrypoint.sh

```diff
@@ -34,6 +27,40 @@ if [[ ! -e /etc/ceph/${CLUSTER}.conf ]]; then
 fi
 }
 
+###############
+# CEPH_DAEMON #
+###############
+
+# If we are given a valid first argument, set the
+# CEPH_DAEMON variable from it
+case "$1" in
+   mds)
+      CEPH_DAEMON=MDS
+      ;;
+   mon)
+      CEPH_DAEMON=MON
+      ;;
+   osd)
```

Comment:
Currently we don't have an `OSD` variable but only `OSD_DIRECTORY` and `OSD_CEPH_DISK`.